app/models/invoice/data_presenter.rb

```diff
@@ -91,6 +91,14 @@ def display_shipment_amount_with_taxes
       Spree::Money.new(shipment.amount + shipment.additional_tax_total, currency:)
     end
 
+    def display_line_item_tax_rate(item)
```

Comment:
It would be good to have a test covering the scenario we are trying to fix

Reply:
I added new tests :)